sktime/forecasting/ardl.py

```diff
@@ -243,6 +247,30 @@ def __init__(
         self.fixed = fixed
         self.causal = causal
         self.trend = trend
+        if self.trend == "ctt":
+            present = _check_soft_dependencies(
```

Comment:
this will now never trigger, since you only check for `statsmodels` and not a particular version.

Please use `_check_soft_dependencies("statsmodels<0.15.0", severity="none")` and use it in both cases.

Prerelease suffixes are included, so 0.15.0 or higher (with prereleae suffix) will not trigger this.